packages/web-components/examples/codesandbox/components-react/expressive-modal/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "ibmdotcom-web-components-react-expressive-modal-example",
+  "version": "0.1.0",
+  "private": true,
+  "description": "Sample project for getting started with the Web Components from Carbon for IBM.com with React.",
+  "license": "Apache-2",
+  "scripts": {
+    "start": "webpack serve"
+  },
+  "dependencies": {
+    "@carbon/ibmdotcom-web-components": "canary",

```

Comment:
@IgnacioBecerra Did we decide to target `latest` instead?

Reply:
One other thing to update in the boilerplate script 👍